docs/fundamentals/week-1/session.md

```diff
@@ -144,7 +144,7 @@ What's useful about thinking about the "who" and the "why"? What could go wrong
 
 The last thing we will do today is ship some code, because we expect you to ship very often at CYF.
 
-You don’t have your shop project yet, so let’s ship your personal home page. In Intro to Digital (ITD) you built a Personal Home Page to complete the course. Now we’ll ship it to become a “real” website. When developers talk about shipping or deployment they mean turning the version that they’re developing into something that users can access.
+You're going to be working on a shop project over the Fundamentals course, but you don’t have that yet, so let’s ship your personal home page. In Intro to Digital (ITD) you built a Personal Home Page to complete the course. Now we’ll ship it to become a “real” website. When developers talk about shipping or deployment they mean turning the version that they’re developing into something that users can access.
```

Comment:
In fact do they do a shop project? I don't think they do - I think it's more open ended. :/ Good catch